app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} must be vaccinated!")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine should be fresh!")

```

Comment:
Nit: fresh is related rather to food than vaccines) You can say that "The vaccine has expired" or something like this)